code/Controllers/CMSMain.php

```diff
@@ -2064,12 +2042,8 @@ public function rollback()
 
     /**
      * Rolls a site back to a given version ID
-     *
-     * @param array $data
-     * @param Form $form
-     * @return HTTPResponse
      */
-    public function doRollback($data, $form)
+    public function doRollback(array $data, Form $form): HTTPResponse
```

Comment:
This method is called by `rollback()` which I think is a form action - and which in any case doesn't pass in a `Form`.

Reply:
The fact that that didn't cause CI to fail is slightly worrying - we might need a new test for rollbacks.

Reply:
Have reverted